### PR TITLE
Adjusted docs

### PR DESCRIPTION
**Pull request by valuta1995**
Signed-off-by: Glenn Visser <gvr340@student.vu.nl>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,20 @@ git clone git@github.com:binpash/pash.git
 cd pash/scripts
 docker build -t "pash/18.04" .
 # Then launch container:
-docker run --name pash-play -it pash/18.04
+```
+
+Then, depending on the OS, run the dockercontainer. Note to change `ubuntu` to `fedora` or `debian` as required.
+
+```sh
+docker run -f ./docker/ubuntu/Dockerfile --name pash-play -it pash:latest
```
Comment (angelhof) [inline README.md:36]: The `-f ` is not supported by `docker run` (at least in my setup). It is only supported by build. Did you intend to add it there?

**Comment by valuta1995** [comment]
Oops you're right. I'll redo it.

⁣Sincerely,
Glenn Visser ​

On 14 Jan 2022, 18:48, at 18:48, Konstantinos Kallas ***@***.***> wrote:
***@***.*** commented on this pull request.

File: README.md (hunk 1 above)
Comment (valuta1995) [reply README.md]: Committed again to this branch. Should be resolved now.

**Comment by nvasilakis** [comment]
Can we test and, if all good, merge? 

(cc @dkarnikis )

**Comment by dkarnikis** [comment]
Changes work on my end. Merging to main

**Pull request merged**